### PR TITLE
expenses: new port

### PR DESCRIPTION
**Pull request by manojkarthick**
#### Description

Simple, Interactive, command line Expense logger.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.3 20D91 x86_64
Xcode 12.5 12E262

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: finance/expenses/Portfile [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                    Answer a series of questions to log your expenses. \
+                    Currently writes to CSV and SQLite backends.
+
+checksums           ${distname}${extract.suffix} \
+                        rmd160  daf975700c0b1aa8a97008d5950425884fc5ae2a \
+                        sha256  e7e1d309a2b2aa83948917532bfdd8dfd8eebf1ce1b5a04ee3b4d14b77c41f5e \
+                        size    35912
+
+build.pre_args-append    -ldflags \
+    '-X github.com/manojkarthick/expenses/cmd.Version=${version}'
```
Comment (harens) [inline finance/expenses/Portfile:24]: nitpick: Just to neaten up the spacing here, I don't think `-append` is required.

File: finance/expenses/Portfile [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/manojkarthick/expenses 0.2.2 v
+categories          finance
+maintainers         {gmail.com:smanojkarthick @manojkarthick} \
+                    openmaintainer
+license             MIT
+
+description         Simple, Interactive, command line Expense logger.
```
Comment (harens) [inline finance/expenses/Portfile:12]: nitpick: I'm not sure whether there's a consensus on this, but generally the `description` doesn't have a full stop whilst the `long_description` does.

**Review by harens — COMMENTED** [review]
Another high quality port for an interesting CLI tool. Thanks @manojkarthick.

Minor nitpicks from me below. If you also wouldn't mind adding a revision field, then it looks good to merge. Thank you for submitting this @manojkarthick 👍

**Comment by manojkarthick** [comment]
Thanks @harens!  - I've made the changes you requested.

**Review by harens — APPROVED** [review]
Thanks @manojkarthick for your contribution! 🎉

**Pull request merged**